### ISSUE TITLE
[Quest] Zen'Kiki and the Cultists - ID 26955 - Western Plaguelands

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#376]
Quest - https://www.wowhead.com/quest=26955/zenkiki-and-the-cultists

**Has same issue as** 
Quest - https://www.wowhead.com/quest=26953/zenkiki-the-druid

- Druid helper isnt assigned
- Quest not complitable without

**Comment by Dranah** [comment]
Same as for the other quest, Zen'kiki is in wrong phase.

**Comment by Bathelmo** [comment]
video please :P?

**Comment by Bathelmo** [comment]
so these are two seperated quests? i am creating a ticket of the other and assigning this to myself

**Comment by Echinaceax** [comment]
> so these are two seperated quests? i am creating a ticket of the other and assigning this to myself

yes two seperate quests , why are you making duplicates @Bathelmo lol?
